### DISCUSSION TITLE
Dev Chat

### DISCUSSION THREAD

**Discussion by caco3** [jomjol/AI-on-the-edge-device#1070] [General] [answered]
@jomjol , @haverland would it make sense to have a developer chat to be able to quicker communicate with each other?

Usually they use something like slack or rocketchat. But also Telegram or so would work.

**Comment by jomjol** [comment, marked as answer]
Yes, good idea. Telegram is fine for me. 

Just send me a short email (see readme) with your contact and I start a channel.

**Reply by ghost** [reply]
I'am using signal. But I can install telegram too

**Reply by caco3** [reply]
I also would have Whatsapp and Threema if that would give a better match :)

**Reply by ghost** [reply]
Now I've installed telegram (iotson)

**Comment by friedpa** [comment]
Ist nur Schade, dass man nun als "Durchschnittsuser" nicht mehr mitbekommt, welche Bugs noch im system sind und an was gearbeitet wird....

**Reply by caco3** [reply]
> Ist nur Schade, dass man nun als "Durchschnittsuser" nicht mehr mitbekommt, welche Bugs noch im system sind und an was gearbeitet wird....

Nein, ich würde weiterhin alles wie weiterhin via issues laufen lassen.
Aber manchmal hat man eine Verständnisfrage oder so, und da macht es wenig Sinn, jeweils ein issue/Diskussion dafür zu eröffnen.

**Reply by ghost** [reply]
> Ist nur Schade, dass man nun als "Durchschnittsuser" nicht mehr mitbekommt, welche Bugs noch im system sind und an was gearbeitet wird....

Da bin ich bei Dir. Deshalb habe ich das Changelog reaktiviert. Ich finde, wir sollten alle closed Issues mit ins Changelog aufnehmen, sofern dafür ein Fix bereit gestellt wurde. Daraus werden ja dann die Releasenotes erzeugt und damit kann der User beim neuen Release besser sehen, ob sein Problem gefixed wurde.
